juniper_official/interface/get-module-temperature.rule

```diff
@@ -0,0 +1,69 @@
+/*
+ * Collect Optical Interface Parameters:
+ *      1) Module Temperature
+ */
+
+healthbot {
+    topic interfaces {
+        rule get-module-temperature {
+            keys [ interface-name ];
+            synopsis "Rule for Optics Anomaly Detection";
+            description "This rule collects optical interface parameters";
+            sensor sensor_optics {
+                open-config {
+                    sensor-name /junos/system/linecard/optics/;
+                    frequency 60s;
+                }
+            }
+
+            field interface-name {
+                sensor sensor_optics {
+                    where "/interfaces/interface/@name =~ /{{interface-name}}/";
+                    path "/interfaces/interface/@name";
+                }
+                type string;
+            }
+
+            field module-temperature {
+                sensor sensor_optics {
+                    path /interfaces/interface/optics/module_temp;
```

Comment:
Seems like this is already covered in https://github.com/Juniper/healthbot-rules/blob/jcloud/juniper_official/interface/check-optical-temp-thresholds.rule